jetson-xavier-seeed-hat.coffee

```diff
@@ -0,0 +1,44 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+BOARD_POWEROFF = 'Remove power from the Jetson Xavier NX Devkit SD-CARD and insert the freshly burnt sd-card.'
+BOARD_POWERON  = 'Power on the board.'
+
+module.exports =
+	version: 1
+	slug: 'jetson-xavier-seeed-hat'
+	aliases: [ 'jetson-xavier-seeed-hat' ]
+	name: 'Nvidia Jetson Xavier NX Devkit with seeed HAT'
+	arch: 'aarch64'
+	state: 'new'
+
```

Comment:
Let's also mark this as a community board maintained by you. This is done by adding "community: true" here.